config.py

```diff
@@ -147,6 +147,7 @@ class Configuration(object):
     S3_SECRET_KEY = "secret_key"
     S3_OPEN_ACCESS_CONTENT_BUCKET = "open_access_content_bucket"
     S3_BOOK_COVERS_BUCKET = "book_covers_bucket"
+    S3_STATIC_FEED_BUCKET = "static_feed_bucket"
```

Comment:
I feel like all the static feed stuff should go into your content server branch, but I could be wrong.